caffe2/opt/custom/fakefp16_transform.cc

```diff
@@ -72,8 +72,8 @@ void fakeFp16Transform(NetDef* net) {
           FLAGS_fake_fp16_conversion_use_fp16_acc,
           FLAGS_fake_fp16_conversion_use_nnpi);
 
-  auto blacklist_pos = glow::ParseNetPositionList(FLAGS_onnxifi_blacklist);
-  auto blacklist_type = glow::ParseBlackListOps(FLAGS_onnxifi_blacklist_ops);
+  auto blocklist_pos = glow::ParseNetPositionList(FLAGS_onnxifi_blocklist);
+  auto blocklist_type = glow::ParseBlockListOps(FLAGS_onnxifi_blocklist_ops);
```

Comment:
`FLAGS_...` are defined elsewhere and those definitions need to be changed too. But those are used all over the place. I would say that this is not a good task for a first-time or external contributor and that you should pick another.

Reply:
i do want to do it if its allowed, i can go and find the header file somewhere in the codebase? 
if not, i will pick another as you suggest sir

Reply:
It's in `glow_net_transform.cc/h`. You can fix all the OSS occurrences, and then we can take of the internal ones.

Reply:
what is oss?
also looks like i was working on those 2 files on separate PR
41784 and 41785

Reply:
Yes, occurrences in open source.

You were, but you need to include all of those changes in one PR to avoid breaking the build.

Reply:
oh i see.. okay i will raise another PR with those 3 files!

Reply:
You can add it to this PR.

Reply:
so i include like fixes {issuenumber} this way?

Reply:
Yes, but you must also include the actual changes.

Reply:
oh, so i got to go back to my branch and edit and push again?

Reply:
Yes.

Reply:
ok noted thanks! im on it.

Reply:
ok just committed with all 3 files now